_inc/client/my-plan/my-plan-header.jsx

```diff
@@ -135,7 +136,12 @@ class MyPlanHeader extends React.Component {
 				);
 				break;
 		}
-		return <div>{ planCard }</div>;
+		return (
+			<>
```

Comment:
Should we give this container a name?

Reply:
This is the [short syntax for React `Fragment`s ](https://reactjs.org/docs/fragments.html#short-syntax). It's equivalent to `<Fragment>`.

Reply:
Good to know, thanks!